modyn/selector/internal/storage_backend/local/local_storage_backend.py

```diff
@@ -1,27 +1,127 @@
 import logging
-from typing import Any, Iterable, Optional
+from pathlib import Path
+from typing import Iterable, Optional
 
+import numpy as np
+from modyn.common.local_storage_backend import LocalStorageBackend as LocalStorageBackendCPP
```

Comment:
Is there a reason we have the LocalStorageBackend in common and then import it here? I am considering either having the CPP wrapper in the storage component or alternatively, both within the same class here. I think it's not a common utility but rather specific for the storage (unlike the triggersamplestorage, where this historaically probably stems from). I'd be happy if you could adjust it in one of those two ways (and move the tests appropriately)

Reply:
done